apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx

```diff
@@ -390,6 +395,14 @@ export const Authenticators: FunctionComponent<AuthenticatorsPropsInterface> = (
                                 }
                                 subHeader={ authenticator.categoryDisplayName }
                                 description={ authenticator.description }
+                                featureStatus={ isSAASDeployment && authenticator
+                                    .defaultAuthenticator?.authenticatorId === AuthenticatorManagementConstants
+                                    .ACTIVE_SESSION_LIMIT_HANDLER_AUTHENTICATOR_ID
+                                    ? (<Chip
+                                        size="small"
+                                        label={ t("common:beta").toUpperCase() }
+                                        className="oxygen-chip-beta"
+                                    />) : null }
```

Comment:
Shall we move this to a seperate function, this is not that readable.

Reply:
done